phpBB/adm/style/acp_profile.html

```diff
@@ -17,7 +17,7 @@
 		</div>
 	<!-- ENDIF -->
 
-	<form id="add_profile_field" method="post" action="{U_ACTION}">
+	<form id="add_profile_field" method="post" action="{U_ACTION}"{S_FORM_ENCTYPE}>
```

Comment:
nothing stops you from doing that. you can use whatever in the backend you wish, but IMO there is no difference between "The only viable option here is to modify the ACTION URL template variable to include enctype: ugly, dangerous and incorrect." and the changes you are proposing.